stage1/init/init.go

```diff
@@ -165,6 +165,10 @@ func installAssets() error {
 	if err != nil {
 		return err
 	}
+	systemdSysusersBin, err := common.LookupPath("systemd-sysusers", os.Getenv("PATH"))
```

Comment:
> hmmm why do we even allow PATH?

What do you mean?

> Should this be with systemd-shutdown, systemd, systemd-journald below?

I put it there because it's something in `usr/bin` and the others are in `/usr/lib/systemd` but I don't have a strong opinion on that.